code/modules/food/recipes_microwave.dm

```diff
@@ -2187,4 +2187,96 @@
 		)
 	result = /obj/item/weapon/reagent_containers/food/snacks/poutine
 
+/*seafood!*/
 
+
+/datum/recipe/salmonsteak
+	reagents = list(SODIUMCHLORIDE = 1, BLACKPEPPER = 1)
+	items = list(/obj/item/weapon/reagent_containers/food/snacks/salmonmeat)
```

Comment:
this might get overridden by the generic meat steak recipe, did you check to see which one the game prioritizes? I have a feeling it's whichever is listed first in the file but I'm not certain